webapp/src/package.ts

```diff
@@ -405,14 +405,14 @@ export class EditorPackage {
     }
 
     savePkgAsync() {
```

Comment:
Notes:
Work to do to work with our cloud stuff?
Switch blobCurrent to cloudCurrent ?

When you create a new project, it currently doesn't sync to the cloud. This might be responsible.. TBD